tests/RobotFramework/tests/tedge/tedge_config_add_remove.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:cli    theme:configuration
+
+
+*** Test Cases ***
+tedge config add/remove value in empty array type config
+    Execute Command    sudo tedge config add c8y.smartrest.templates 1
+    ${add}    Execute Command    tedge config list
+    Should Contain    ${add}    c8y.smartrest.templates=["1"]
+
+    Execute Command    sudo tedge config remove c8y.smartrest.templates 1
+    ${remove}    Execute Command    tedge config list
+    Should Contain    ${remove}    c8y.smartrest.templates=[]
+
```

Comment:
I would suggest checking the initial state in order to ensure consistency and to validate that the setup is correct, something like:
```
tedge config add/remove value in empty array type config
    # Verify initial state
    ${initial}    Execute Command    tedge config list
    Should Not Contain    ${initial}    c8y.smartrest.templates

    # Add value to empty array
    Execute Command    sudo tedge config add c8y.smartrest.templates 1
    ${add}    Execute Command    tedge config list
    Should Contain    ${add}    c8y.smartrest.templates=["1"]

    # Remove value
    Execute Command    sudo tedge config remove c8y.smartrest.templates 1
    ${remove}    Execute Command    tedge config list
    Should Contain    ${remove}    c8y.smartrest.templates=[]